dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/AccessTokenService.java

```diff
@@ -55,7 +55,7 @@ public interface AccessTokenService {
      * @param token token string (if it is absent, it will be automatically generated)
      * @return create result code
      */
-    Map<String, Object> createToken(User loginUser, int userId, String expireTime, String token);
+    Result createToken(User loginUser, int userId, String expireTime, String token);
```

Comment:
```suggestion
    CreateTokenResponse createToken(User loginUser, int userId, String expireTime, String token);
```
I think use the specific response object is better, WDYT?

Reply:
I think we should not do this in this revision since there are many others api‘s service's return params is very complicated. Changing this may cause unpredictable bugs in ```v1``` apis. @caishunfeng

Reply:
Yeah, I found out it is the V1 entry point, we should better keep in the old method to avoid addition error

Reply:
OK, I get your point.